rust/bin-shared/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "firezone-bin-shared"
+version = "0.1.0"
+edition = "2021"
+description = "Firezone-specific modules shared between binaries."
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+url = { version = "2.3.1", default-features = false }
+tracing-subscriber = { workspace = true, features = ["env-filter"] }
+tracing = { workspace = true }
+tracing-log = "0.2"
+clap = { version = "4.5", features = ["derive",  "env"] }
+ip_network = { version = "0.4", default-features = false, features = ["serde"] }
+anyhow = "1.0.82"
+futures = "0.3"
+tokio = { workspace = true, features = ["rt"] }
+connlib-shared = { workspace = true }
```

Comment:
I've been lazily alphabetizing deps as I see them so that it's easy to look them up in the files

Reply:
Should we add https://crates.io/crates/cargo-sort to the repo?

Reply:
Unfortunately, the actual rustfmt issue for this is pretty stale: https://github.com/rust-lang/rustfmt/issues/4091

Reply:
Yeah cargo-sort would be okay. I'm surprised this isn't common practice, with enough deps a linear search of "do I already have this?" can get slow

Reply:
Done: https://github.com/firezone/firezone/pull/5851.